CONTRIBUTING.md

```diff
@@ -0,0 +1,83 @@
+# Contributing
+
+Before contributing to this repository, discuss the changes through an issue,
+email, or any mode of communciation.
+
+Follow the code of conduct follow in all interactions with the project.
+
+### Reporting Bugs
+
+Bugs are tracked as [GitHub issues](https://guides.github.com/features/issues/).
+
+1. Create an issue and provide the required information in the [issue template](ISSUE_TEMPLATE.md).
+2. Describe the problem with the following details:
+  - A self-explanatory title.
+  - Steps to reproduce the problem. Provide the steps with details such as, what is the step and how is it done.
+  - Examples to demonstrate the steps. Include links to files or reusable snippets that are used in the examples. For providing snippets in the issue, use [Markdown code blocks](https://help.github.com/articles/markdown-basics/#multiple-lines).
+  - Describe the result you get when you follow the steps, and state the issues, if any.
+  - Explain the expected result and the reason.
+  - Add screenshots of the results of each step.
+3. Add the following configuration and environment details:
+  - Versions of Node and NPM that you use. Run the  `node --version` and `npm --version` command in the terminal to get the correct version.
+  - Name version of the operating system that you use.
+  - Packages installed on your machine. Review your `package.json` file to get the list. Ensure that new packages are installed successfully during development.
+  - Properties that you add to the application. The `ep.config.json` file  provides the contents of the file, in a [code block](https://help.github.com/articles/markdown-basics/#multiple-lines) or with a link to a [gist](https://gist.github.com/).
```

Comment:
I think that there is an extra space between "file" and "provides" in the second sentence.